po/it.po

```diff
@@ -0,0 +1,206 @@
+# ITALIAN TRANSLATION FOR WHALER.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.sdv43.whaler package.
+# ALBANO BATTISTELLA <ALBANO_BATTISTELLA@HOTMAIL.COM>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.sdv43.whaler\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-05-11 20:59+0300\n"
+"PO-Revision-Date: 2022-05-15 12:26+0100\n"
+"Last-Translator: Albano Battistella <albano_battistella@hotmail.com>\n"
+"Language-Team: Italian <LL@li.org>\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/com.github.sdv43.whaler.desktop.in:3
+#: data/com.github.sdv43.whaler.appdata.xml.in:4
+msgid "Whaler"
+msgstr "Whaler"
+
+#: data/com.github.sdv43.whaler.desktop.in:4
+msgid "Docker Client"
+msgstr "Docker Client"
+
+#: data/com.github.sdv43.whaler.desktop.in:5
+msgid "Manage your Docker containers"
+msgstr "Gestisci i tuoi contenitori Docker"
+
+#: data/com.github.sdv43.whaler.desktop.in:8
+msgid "com.github.sdv43.whaler"
+msgstr "com.github.sdv43.whaler"
+
+#: data/com.github.sdv43.whaler.desktop.in:11
+msgid "Docker;Virtualizaton;Development;"
+msgstr "Docker;Virtualizzazione;Sviluppo;"
+
+#: data/com.github.sdv43.whaler.appdata.xml.in:5
+msgid "Docker Container Management"
+msgstr "Gestione dei container Docker"
+
+#: data/com.github.sdv43.whaler.appdata.xml.in:8
+msgid "Selivestrov Dmitriy"
+msgstr "Selivestrov Dmitriy"
+
+#: data/com.github.sdv43.whaler.appdata.xml.in:10
+msgid ""
+"Whaler provides basic functionality for managing Docker containers. The app "
+"can start and stop both standalone containers and docker-compose "
+"applications. Also, it supports viewing container logs."
+msgstr ""
+"Whaler fornisce funzionalità di base per la gestione dei container Docker. L'app "
+"può avviare e interrompere sia i contenitori standalone che le applicazioni "
+"docker-compose. Inoltre, supporta la visualizzazione dei registri del container."
+
+#: data/com.github.sdv43.whaler.appdata.xml.in:15
+msgid ""
+"The solution is perfect for those who are looking for a simple tool to "
+"perform some basic actions. For the app to run correctly, make sure that "
+"Docker and docker-compose are installed on your system."
+msgstr ""
+"La soluzione è perfetta per chi cerca un semplice strumento per "
+"eseguire alcune azioni di base. Affinché l'app funzioni correttamente, assicurati che "
+"Docker e docker-compose siano installati sul tuo sistema."
+
+#: src/Utils/Sorting/SortingName.vala:12
+msgid "Name"
+msgstr "Nome"
+
+#: src/Utils/Sorting/SortingType.vala:10
+msgid "Type"
+msgstr "Tipo"
+
+#: src/State/Root.vala:40
+msgid "Cannot get list of docker containers"
+msgstr "Impossibile ottenere l'elenco dei contenitori Docker"
+
+#: src/State/Root.vala:48
+msgid "Docker isn't installed"
+msgstr "Docker non è installato"
```

Comment:
Hi, thank you for your PR.

This line contains a bit confusing msgid which I'm going to change later. The correct meaning in English is given [here](https://github.com/sdv43/whaler/blob/383b3cca1802cb2e6c095e4a11fd126733ac8162/po/en.po#L82). Could you please translate the current English version?